modules/ROOT/pages/amazon/amazon-sqs-connector-xml-maven.adoc

```diff
@@ -0,0 +1,72 @@
+= Amazon SQS XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to Amazon SQS Connector in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.
+
+[NOTE]
+====
+You can avoid the manual process described in this topic by using Studio. When you install Amazon SQS Connector in Studio, Studio automatically populates the XML code with the Amazon SQS Connector namespace and schema location, and adds a dependency to the `pom.xml` file.
+====
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/amazon-sqs
+http://www.mulesoft.org/schema/mule/amazon-sqs/current/mule-amazon-sqs.xsd
+----
+
+This example shows how the namespace statements are placed in the XML:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+xmlns:http="http://www.mulesoft.org/schema/mule/http"
+xmlns:sqs="http://www.mulesoft.org/schema/mule/sqs"
+xmlns="http://www.mulesoft.org/schema/mule/core"
+xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/sqs
+http://www.mulesoft.org/schema/mule/sqs/current/mule-sqs.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+----
+
+== Add a POM File Dependency
+
+The `pom.xml` Maven file generated by Studio contains dependencies for
+a Mule app. When you code a Mule app manually, include this XML snippet in
+your `pom.xml` file to enable access to the connector.
+
+The Mule runtime engine translates the `RELEASE` dependency keyword to the current version.
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>com.mulesoft.connectors</groupId>
+    <artifactId>mule-amazon-sqs-connector</artifactId>
+    <version>RELEASE</version>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+To obtain the most up-to-date `pom.xml` file information, access the
+connector in https://www.mulesoft.com/exchange/[Anypoint Exchange]
+and click *Dependency Snippets*.
```

Comment:
Is this a prerequisite to "include this XML snippet in your..."?
Should it come earlier in the topic?

Reply:
Noted for next template update.